### PR TITLE
Release 1.5.1 and NEWS.md file

### LINKED ISSUE #14: Add a NEWS.md file
Similar to other HADES repos (e.g. https://github.com/OHDSI/SqlRender/blob/master/NEWS.md) . This will automatically be included in the package website when running pkgdown.

### PR DESCRIPTION
**Pull request by azimov**
* closes #14 
* Package Maintenance
* Docs build to pdf and added

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+SelfControlledCohort 1.6.0
+==========================
+- Added changelog in NEWS.md
```
Comment (schuemie) [inline NEWS.md]: Please distinguish between 'Changes' and 'Bug fixes'. (See for example [here](https://github.com/OHDSI/Andromeda/blob/master/NEWS.md))

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- Added support for github actions and removed travis
```
Comment (schuemie) [inline NEWS.md]: The NEWS.md file is for users to understand what changed for them. (others can just look at the commit history). I would argue the users do not need to know about this change. If you think they do, you will need to add a lot more explanation of why this is relevant for them.

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- Added computeTarDistribution flag, which computes the distribution of exposure time
```
Comment (schuemie) [inline NEWS.md]: To which function?

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+ windows for subjects experiencing the outcome and average absolute time between
+ exposure and outcome.
+ - Added guide for usage with some examples
```
Comment (schuemie) [inline NEWS.md]: I think you're referring to the vignette? I would use the word 'vignette', which should be well know to R users.

File: NEWS.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+ - Fixed issue with DatabaseConnector v 4.0.0 connectionDetails object on osx and linux
```
Comment (schuemie) [inline NEWS.md]: A good example of what would go under 'Bug fixes'.

File: NEWS.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+ - Updated author in DESCRIPTION file
```
Comment (schuemie) [inline NEWS.md]: Again, not something users would care about.

**Comment by azimov** [comment]
@schuemie if you're happy with this PR I will merge develop to master for the release.

**Review by schuemie — APPROVED** [review]

**Comment by schuemie** [comment]
Looks good! Please merge

**Pull request merged**